packages/quantic/force-app/main/default/lwc/quanticDocumentSuggestion/quanticDocumentSuggestion.html

```diff
@@ -0,0 +1,41 @@
+<template>
+  <c-quantic-search-interface engine-id={searchEngineId}>
+    <template if:true={loading}>
+      <div class="loading-holder">
+        <lightning-spinner alternative-text={labels.loading}></lightning-spinner>
+      </div>
+    </template>
+    <template if:false={loading}>
+      <template if:true={hasSuggestions}>
```

Comment:
While it wasn't defined in the mock-ups we should render something when no suggestions are returned. @adroletCoveo has been made aware of the situation and added a note in the [figma file](https://www.figma.com/file/mGZNYe9DbqIL1NO4m292PG/Case-Assist-User-experience?node-id=348%3A30935).